java/java.hints/test/unit/src/org/netbeans/modules/java/hints/OrganizeMembersTest.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.netbeans.modules.java.hints;
+
+import java.util.prefs.Preferences;
+import org.netbeans.api.editor.mimelookup.MimeLookup;
+import org.netbeans.junit.NbTestCase;
+import org.netbeans.modules.editor.java.JavaKit;
+import org.netbeans.modules.java.hints.test.api.HintTest;
+import org.netbeans.modules.java.ui.FmtOptions;
+import org.openide.util.NbBundle;
+
+/**
+ * @author mbien
+ */
+public class OrganizeMembersTest extends NbTestCase {
+    
+    public OrganizeMembersTest(String name) {
+        super(name);
+    }
+    
+    @Override
+    protected void setUp() throws Exception {
+        super.setUp();
+        HintTest.create(); // initializes lookup (stolen from bugs.TinyTest)
+        MimeLookup.getLookup(JavaKit.JAVA_MIME_TYPE)
+                  .lookup(Preferences.class)
+                  .putBoolean(FmtOptions.sortMembersInGroups, true);
+    }
+
+    @Override
+    protected boolean runInEQ() {
+        // without it, hint markers are sometimes not found
+        return true;
+    }
```

Comment:
don't know why this is needed. `OrganizeMembers` invokes `runModificationTask` which calls `copy.rewrite()` at some point. 

`result.getDifferences(source.getFileObject())` is sometimes null if not on EDT.